docs/rules/comma-dangle.md

```diff
@@ -12,6 +12,26 @@ var foo = {
 ```
 
 On the other hand, trailing commas simplify adding and removing items to objects and arrays, since only the lines you are modifying must be touched.
+Additionally, another argument in favor of trailing commas is that it improves the clarity of diffs when an item is added or removed from an object or array:
```

Comment:
I believe the original sentence was more focused on the argument that editing an object or array literal is simplified since you only have to e.g. delete the line with the item you want to delete, and not have to move your cursor up to the previous line to remove the comma. I think that would get lost if we rewrite it as per your suggestion.
But I see your point and of course I can rewrite it to a single sentence if you still prefer!